cmake/modules/SanitizerFlags.cmake

```diff
@@ -41,6 +41,8 @@ function(enable_sanitizers target)
     )
     target_link_libraries(${target} PRIVATE
       "$<$<CXX_COMPILER_ID:MSVC>:clang_rt.asan_dynamic-x86_64;clang_rt.asan_dynamic_runtime_thunk-x86_64>"
```

Comment:
This and other lines hardcoding the link to some support libraries when compiling against 16.9 version of VS2019 is failing for the latest release.
Are you sure you need them ?

Reply:
I changed it a bit. Does it now work for you?

Reply:
It works fine for me on Windows and would detect an out of bounds memory access error. Congratulations !